packages/utils/Readme.md

```diff
@@ -3,3 +3,119 @@
 This package contains various uncategorized tools. It should be thought of
 as the shed in your back yard where you put the stuff that doesn't belong
 elsewhere. There isn't an overarching theme for the items in this package.
+
+## Calls module
+This module contains traits used to call another contract.  Do not forget to add the `use` statement for the traits you want.
+```rust
+use secret_toolkit::utils::{InitCallback, HandleCallback, Query};
+```
+Also, don't forget to add the toolkit dependency to your Cargo.toml
+### Instantiating another contract
+If you want to instantiate another contract, you should first copy/paste the InitMsg of that contract.  For example, if you wanted to create an instance of the counter contract at https://github.com/enigmampc/secret-template
```

Comment:
I would add an empty line before headings, just to make it easier to read the source code